### PR TITLE
ISPN-10512 CLI User Guide

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.jboss.org/browse/ISPN-10512

### CODE REVIEW

**Comment by wburns** [comment]
Someone has been busy :)
So looking at this I have to admit is there some way we can source this doc with the help/man pages that the CLI uses. @tristantarrant maybe you might have an idea?

**Comment by oraNod** [comment]
@wburns check out https://raw.githubusercontent.com/infinispan/infinispan/512b78f0433370c856b8809c0a5880b6ba7e8199/documentation/src/main/asciidoc/stories/assembly_cli_command_reference.adoc 

This uses the adoc files that Tristan created for man pages. I'm working on a script to generate the reference so docs can automatically pick up the contents of the help resouces directory.

**Comment by wburns** [comment]
Cool, yeah that is nice :)

File: documentation/src/main/asciidoc/topics/proc_cli_adding_entries.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+[//containers/default]> put --cache=mycache hello world
+----
++
+[TIP]
+====
+If the CLI is in the context of a cache, do `put k1 v1` for example:
+
+----
+[//containers/default]> cd mycache
+[//containers/default/mycache]> put hello world
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/proc_cli_adding_entries.adoc]: The context for the cache would be //containers/default/caches/mycache

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**